routes/voyager.php

```diff
@@ -47,7 +47,7 @@
                 Route::get($dataType->slug.'/{id}/restore', $breadController.'@restore')->name($dataType->slug.'.restore');
                 Route::get($dataType->slug.'/relation', $breadController.'@relation')->name($dataType->slug.'.relation');
                 Route::post($dataType->slug.'/remove', $breadController.'@remove_media')->name($dataType->slug.'.media.remove');
-                Route::resource($dataType->slug, $breadController);
+                Route::resource($dataType->slug, $breadController, ['parameters' => [$dataType->slug => 'id']]);
```

Comment:
Ah, nevermind... I see.  You're forcing **all** resource route parameters to be `id`

Reply:
Yes, exactly. 
We only have one parameter so not really any reason to not use just {id} if it solves the utf8 character problem.